sql/fxa_users_last_seen_v1.sql

```diff
@@ -0,0 +1,47 @@
+WITH
+  current_sample AS (
+  SELECT
+    -- Include dummy dates for date_last_* fields to make schema match with `previous`.
+    DATE '2000-01-01' AS date_last_seen,
+    DATE '2000-01-01' AS date_last_seen_in_tier1_country,
```

Comment:
I think we should use `DATE(NULL)` here if it is a dummy column

Reply:
Makes sense. I didn't know DATE(NULL) was possible.